masterlist.yaml

```
@@ -23577,6 +23577,108 @@ plugins:
       - <<: *alreadyInOrFixedByX
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp")'
+    
+    ### Dawn of Skyrim - Modern Patches ###
+  - name: 'DoS_MP_Whiterun.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/65467/' ]
```

You can combine the locations using a regex plugin, & you can also add a name to it so it can be used in the UI:

```suggestion
  - name: 'DoS_MP_Whiterun.*\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65467/'
        name: 'Dawn of Skyrim - Modern Patches'
  - name: 'DoS_MP_Whiterun.esp'
```